web/src/cypress/support/schedule.ts

```diff
@@ -138,6 +145,56 @@ function deleteSchedule(id: string): Cypress.Chainable<void> {
   })
 }
 
+function createTemporarySchedule(
+  scheduleID?: string,
+  options?: Partial<TemporarySchedule> & { scheduleID?: string },
+): Cypress.Chainable<void> {
+  const mutation = `
+    mutation($input: SetTemporaryScheduleInput!) {
+      setTemporarySchedule(input: $input)
+    }
+  `
+
+  if (!scheduleID) {
+    return cy
+      .createSchedule()
+      .then((s: Schedule) => createTemporarySchedule(s.id, options))
+  }
+
+  const nowDT = DateTime.local()
+  const input = options || {}
+  input.scheduleID = scheduleID
+
+  // set start to start of today or 7 days before set end
+  if (!input.start && !input.end) {
+    input.start = nowDT.startOf('day').toISO()
+  } else if (!input.start && input.end) {
+    input.start = DateTime.fromISO(input.end).minus({ days: 7 }).toISO()
```

Comment:
We should randomize this a little so tests don't depend on consistency (i.e. if they need consistency, then they should be explicit)